retrofit/build.gradle

```diff
@@ -34,18 +34,15 @@ android
 
 dependencies
 {
-  //Smart&Soft
-  api ("com.smartnsoft:droid4me:2.8.0")
-  implementation ("com.smartnsoft:droid4me.ext:2.7.0")
   implementation ("org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version")
+  api("com.android.support:appcompat-v7:27.1.1")
```

Comment:
You should update to appcompat 28.0.0. If not, it might cause some compatibility issues